src/modules/rest/impl/src/main/java/it/geosolutions/geostore/services/rest/security/oauth2/google/OAuthGoogleSecurityConfiguration.java

```diff
@@ -70,7 +70,7 @@ public OAuth2Configuration configuration() {
      */
     @Override
     @Bean(value = "googleOpenIdRestTemplate")
-    @Scope(value = "session", proxyMode = ScopedProxyMode.TARGET_CLASS)
+    @Scope(value = "request", proxyMode = ScopedProxyMode.TARGET_CLASS)
```

Comment:
Is this something related to KeyCloak? are we sure it won't break any other oauth2 integration?

Reply:
the scope value session is an error placed that by me on a previous commit. It might endup creating an HttpSession to store the bean in a stateless context like the geostore rest api. In the context of the Openid filter usage of the above bean it will not change anything regarding the functioning of it, since spring will provide the bean anyway bound to the request instead that bound to the session. I've test locally the google open id authentication after the modification.